app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearMainView.kt

```diff
@@ -58,11 +58,13 @@ class SettingsWearMainView : AppCompatActivity() {
     }
 
     private fun onOnboardingComplete(result: ActivityResult) {
-        val intent = result.data!!
-        val url = intent.getStringExtra("URL").toString()
-        val authCode = intent.getStringExtra("AuthCode").toString()
-        val deviceName = intent.getStringExtra("DeviceName").toString()
-        val deviceTrackingEnabled = intent.getBooleanExtra("LocationTracking", false)
-        settingsWearViewModel.sendAuthToWear(url, authCode, deviceName, deviceTrackingEnabled)
+        if (result.data != null) {
```

Comment:
If that's null we should log something otherwise it'll be confusing when nothing happens.

Reply:
yea good call :)